app/clients/sms/firetext.py

```diff
@@ -62,11 +61,27 @@ def init_app(self, current_app, statsd_client, *args, **kwargs):
         self.api_key = current_app.config.get('FIRETEXT_API_KEY')
         self.from_number = current_app.config.get('FROM_NUMBER')
         self.name = 'firetext'
+        self.url = "https://www.firetext.co.uk/api/sendsms/json"
         self.statsd_client = statsd_client
 
     def get_name(self):
         return self.name
 
+    def record_outcome(self, success, response):
+        log_message = "API {} request {} on {} response status_code {}".format(
+            "POST",
+            "succeeded" if success else "failed",
+            self.url,
+            response.status_code
+        )
+
+        if not success:
```

Comment:
It's easier to read when if statements are written as positive conditions "if successs then ..." rather than "if not success then .."